azero/scripts/bridge_weth.ts

```diff
@@ -0,0 +1,67 @@
+import { ApiPromise, WsProvider, Keyring } from "@polkadot/api";
```

Comment:
Isn't that a similar thing to what e2e-tests `azero_to_eth` is doing?

Reply:
similar but not the same. This is a make target for quickly spawning requests when manually testing, I find it very usefull